Core/StatisticsUserDefaults.swift

```diff
@@ -25,7 +25,7 @@ public class StatisticsUserDefaults: StatisticsStore {
     private let groupName: String
     
     private struct Keys {
-        static let cohortVersion = "com.duckduckgo.statistics.cohortVersion.key"
+        static let cohortVersion = "com.duckduckgo.statistics.cohortVersion.v2.key"
```

Comment:
This is because people already may have the old one saved?

Reply:
This is an interesting case. Hard to test programatically?

Reply:
The behaviour is already tested e.g that the key works. What's not tested is the migration - I generally only migration test release versions. If this had been released I would have updated their cohort rather than just blowing it away and migration tested it.